demos/music/assets/img_lv_demo_music_cover_1.c

```diff
@@ -184,7 +184,7 @@ const LV_ATTRIBUTE_MEM_ALIGN uint8_t img_lv_demo_music_cover_1_map[] = {
 };
 
-lv_img_dsc_t img_lv_demo_music_cover_1 = {
+lv_image_dsc_t img_lv_demo_music_cover_1 = {
```

Comment:
It should be a const var. Let's correct it.

```suggestion
const lv_image_dsc_t img_lv_demo_music_cover_1 = {
```